data/epub.css

```diff
@@ -23,3 +23,8 @@ span.underline{ text-decoration: underline; }
 q { quotes: "“" "”" "‘" "’"; }
 div.column{ display: inline-block; vertical-align: top; width: 50%; }
 div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
+/* Apple Books 2.4+ doesn't like overflow:auto on syntax highlighting generated by skylighting */
```

Comment:
I noticed a lot of similar CSS rules in the final, generated epub CSS with hacks for kindle, etc. Is that skylighting generating that? Should we, instead, fix this there?

Reply:
Yes.  skylighting adds this in the CSS:
``` css
@media screen {
 div.sourceCode { overflow: auto; }
}
```
Code is in skylighting-core, Skylighting.Format.HTML
The problem is that we *want* the behavior of overflow: auto, not overflow:visible, which causes long code lines to overflow the box.  Changing this would not be desirable for many users (and note that the same code is used for regular HTML).

Reply:
If you can override it in your own epub.css, that's good to know.
Have you tried?

Reply:
Yup! Works fine.